charts/athens-proxy/README.md

```diff
@@ -25,14 +27,101 @@ $ helm repo add athens https://gomods.github.io/athens-charts
 $ helm repo update
 ```
 
-Next, install the chart with default values to `athens` namespace:  
+Next, install the chart with default values to `athens` namespace: 
 
 ```
 $ helm install athens/athens-proxy -n athens --namespace athens
 ```
 
 This will deploy a single Athens instance in the `athens` namespace with `disk` storage enabled. Additionally, a `ClusterIP` service will be created.
 
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling |
```

Comment:
I'd recommend using a tool like [Markdown Table Formatter](http://markdowntable.com/) since all rows have different lengths here

Reply:
Resolving this as it comes from the template/helm-docs